### DISCUSSION TITLE
What was the "aha" moment for you?

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/code-server#5116] [General]
What was the "aha" moment where code-server clicked for you?

**Comment by jsjoeio** [comment]
For me, it was realizing that having VS Code in the browser meant I could access my dev environment from any device with a browser 🤯 i.e. being able to code from my friend's laptop, my iPad or even my phone.

**Comment by lg-nineoneninethreenine** [comment]
For me, it just meant use the browser on any crappy laptop you want with the same devcenter open. And also - it's lightweight, so it doesn't load my PC up. Only thing missing is a open/closed ports control center. That'd be great for a person who wants to go to noVNC easily.

**Comment by lg-nineoneninethreenine** [comment]
Moreover, it works headless, so I can start code-server on any device that doesn't have a display. Maybe if I am lucky, it might work on an Amazon Kindle(that would be hilarious).
